controllers/actions.github.com/resourcebuilder.go

```diff
@@ -311,6 +311,8 @@ func mergeListenerPodWithTemplate(pod *corev1.Pod, tmpl *corev1.PodTemplateSpec)
 		switch c.Name {
 		case "listener":
```

Comment:
Great catch! Can you remove this case because I mistakenly wrote that the container name for the listener is the `listener`? We don't really need this case anymore